### DISCUSSION TITLE
Remote for Canada Shaw Cable (based on Xfinity hardware)

### DISCUSSION THREAD

**Discussion by blasco403** [GerryDazoo/Slinger#147] [General]
[shawcable(xfinity)comcastX1(XR5)remote.txt](https://github.com/GerryDazoo/Slinger/files/9814428/shawcable.xfinity.comcastX1.XR5.remote.txt)

Custom Remote for Shaw Cable Systems (based on Xfinity Hardware).

Nothing Fancy, button locations as per remote.

Tested with Singbox 500 and 350.

**********************************************

Updated with A,B,C,D buttons; 3 min skip ahead, 1 min skip back, and one sample button that tunes to a specific station.

[shawcable(xfinity)comcastX1(XR5)remote.txt](https://github.com/GerryDazoo/Slinger/files/9854310/shawcable.xfinity.comcastX1.XR5.remote.txt)

**Comment by stansoltz** [comment]
Thanks.  I assume this could work with Rogers Ignite IPTV in Canada, also based on Comcast Xfinity platform?

**Comment by blasco403** [comment]
Not sure, you would have to try it to find out. 

TTYL

Ron

On Oct 21, 2022, at 4:38 PM, stansoltz ***@***.***> wrote:

﻿
Thanks. I assume this could work with Rogers Ignite IPTV in Canada, also based on Comcast Xfinity platform?
